generator/graphql-kotlin-schema-generator/build.gradle.kts

```
@@ -11,7 +11,7 @@ val slf4jVersion: String by project
 dependencies {
     api("com.graphql-java:graphql-java") {
         version {
-            strictly(graphQLJavaVersion)
+            strictly("[$graphQLJavaVersion, 18.0)")
```

I think it should be 
```suggestion
            strictly("[$graphQLJavaVersion, 18.0[")
```

---

I actually don't think we should use `[`

> An upper bound exclude acts as a prefix exclude. This means that `[1.0, 2.0[` will also exclude all versions starting with 2.0 that are smaller than 2.0. For example versions like 2.0-dev1 or 2.0-SNAPSHOT are no longer included in the range.

https://docs.gradle.org/current/userguide/single_versions.html